cmd/installer/cli/install.go

```diff
@@ -153,6 +161,85 @@ func InstallCmd(ctx context.Context, name string) *cobra.Command {
 	return cmd
 }
 
+func installCmdExample() string {
+	if os.Getenv("ENABLE_V3") != "1" {
+		return ""
+	}
+
+	return `
+  # Install on a Linux host
+  replicated install \
```

Comment:
maybe this is just a placeholder for now, but this should use the app slug rather than `replicated`. same with other example.